app/controllers/account/chrome_extensions_controller.rb

```diff
@@ -0,0 +1,31 @@
+class Account::ChromeExtensionsController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+    @page = Page.new(source_url: params[:source_url], title: params[:title])
+  end
+
+  def create
+    @company = current_user.companies.find_by(id: params[:page][:company_id])
+    @page = Page.new(params_without_company_id)
+    @page.company = @company
+    add_metadata_to @page
```

Comment:
remove this method
and put here 
+    page.page_type = Page::CHROME_EXTENSION
 +    page.status = Page::PENDING_STATUS